package configDB;

import java.security.PublicKey;
import java.sql.SQLException;
import java.util.ArrayList;

import login.Validate;
import operate.Adpater;

public class Main {

	public static void main(String args[]) throws SQLException {
		//JDBC jdbc = new JDBC("localhost","3306","libaryDB","root","12345678");

//		ArrayList<ArrayList<String>> result = JDBC.readData("book_detail");
//		System.out.println(result);
//
//		
//		result = JDBC.readBy("Book", "dueDate","");
//		System.out.println(result.size());
//		
//		result = JDBC.readBys("Book", "status", "2", "dueDate", "2020-02-02");
//		System.out.println(result);
//
//		int c = JDBC.getCountBy("Book","status","2","dueDate","2020-02-02");
//		System.out.println(c);
		
		//need to call this two method to insert date to DB and follow the format
		//JDBC.setKey("name");
		//JDBC.insert("role","ROLE_MEMBER");
//		jdbc.delete("book_detail", "code", "004");
		//System.out.println("Hello");
		//Validate.sendMail("hengta56@gmail.com");
//		ArrayList<ArrayList<String>> user = JDBC.readBy("secuser", "username", "liza");
		
//		user.toArray();
		
//		System.out.println(user.get(0));
		
		
	
		
	}
	
	
	

}
